test/cluster/blue-green-deployment/setup.td

```diff
@@ -0,0 +1,95 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ postgres-execute connection=postgres://mz_system:materialize@${testdrive.materialize-internal-sql-addr}
+ALTER SYSTEM SET enable_unmanaged_cluster_replicas = true;
+
+> DROP CLUSTER IF EXISTS prod CASCADE
+> DROP CLUSTER IF EXISTS prod_deploy CASCADE
+> CREATE CLUSTER prod REPLICAS (replica1 (
+  STORAGECTL ADDRESSES ['clusterd1:2100'],
+  STORAGE ADDRESSES ['clusterd1:2103'],
+  COMPUTECTL ADDRESSES ['clusterd1:2101'],
+  COMPUTE ADDRESSES ['clusterd1:2102'],
+  WORKERS 1))
```

Comment:
Would more workers and/or more storageds make this test more stessful/realistic?

For example, have the two clusters being replaced have different sizes/shapes?

Reply:
I'm not sure. My understanding is that one of the motivations is also changing the schema without any downtime. I'll try a different cluster though.